Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -1196,6 +1196,12 @@ int CLuaElementDefs::GetAttachedElements(lua_State* luaVM)
             {
                 CClientEntity* pAttached = pEntity->GetAttachedEntity(i);
                 assert(pAttached->GetAttachedTo() == pEntity);
+                if (IS_MARKER(pEntity))
+                {
+                    CClientMarker* marker = static_cast<CClientMarker*>(pEntity);
```

Comment:
Can we move this code out of the loop? 


Above the loop:

```
CClientEntity* pIgnoredElement = nullptr;
if (IS_MARKER(pEntity))
{
    CClientMarker* marker = static_cast<CClientMarker*>(pEntity);
    pIgnoredElement = marker->GetColShape();
}
``

Inside the loop:

```
if (pAttached == pIgnoredElement)
    continue;
```

Reply:
Although I have removed the code now, I don't think the compiler will optimize this. What you suggested is good.